laws/src/main/scala/cats/laws/MonadErrorLaws.scala

```diff
@@ -19,6 +19,12 @@ trait MonadErrorLaws[F[_], E] extends ApplicativeErrorLaws[F, E] with MonadLaws[
 
   def redeemWithDerivedFromAttemptFlatMap[A, B](fa: F[A], fe: E => F[B], fs: A => F[B]): IsEq[F[B]] =
     F.redeemWith(fa)(fe, fs) <-> F.flatMap(F.attempt(fa))(_.fold(fe, fs))
+
+  override def adaptErrorPure[A](a: A, f: E => E): IsEq[F[A]] =
+    F.adaptError(F.pure(a)) { case x => f(x) } <-> F.pure(a)
+
+  override def adaptErrorRaise[A](e: E, f: E => E): IsEq[F[A]] =
+    F.adaptError(F.raiseError[A](e)) { case x => f(x) } <-> F.raiseError(f(e))
```

Comment:
hmmm, do you mind to comment here on how this fixes BC? not quite obvious to me.

Reply:
It's the same general problem and fix as in #3162—when you move a method from a trait to a super-trait, you no longer get the synthetic static method (the bridge method or forwarder or whatever) on the lower trait, which can cause problems in real code. Including an override in the lower trait fixes the issue.

I can add a comment in the code when I'm back at my desktop.